workflow-templates/README.md

```diff
@@ -2,9 +2,11 @@
 
 ## Pull Request Deploy Previews
 
+Note: instructions below were written for the Quarto Deploy Preview workflow template but apply the same to the Pkgdown Deploy Preview workflow template.
+
 Files:
-- deploy-preview.yml
-- deploy-preview.properties.json
+- deploy-preview.yml/deploy-preview-pkgdown.yml
+- deploy-preview.properties.json/deploy-preview-pkgdown.properties.json
```

Comment:
So `deploy-preview.yml/` and `deploy-preview.properties.json/` are directories?

Reply:
sorry, no just meant the slash as "or", so deploy-preview.yml OR deploy-preview-pkgdown.yml

Reply:
Ah that makes way more sense. Can it say "or" instead?

Reply:
Sure